bitcoin.stackexchange.com/main.py

```diff
@@ -1,19 +1,19 @@
 import os
+import sys
 import time
 from datetime import datetime
-from dotenv import load_dotenv
 from loguru import logger
 from tqdm import tqdm
+
 from utils import download_dump, extract_dump, parse_posts, parse_users, strip_tags, document_view, document_add
 import traceback
-load_dotenv()
-
 
-if __name__ == "__main__":
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-    INDEX = os.getenv("INDEX")
+from config.conf import INDEX_NAME, DATA_DIR
 
-    BASE_DIR = os.getenv("DATA_DIR", ".")
+if __name__ == "__main__":
+    BASE_DIR = os.getenv(DATA_DIR, ".")
```

Comment:
you can also import `BASE_DIR` from `config.conf`.

Additionally, if you wanted to do the config a bit more flexible you could structure it like [this one here](https://github.com/kouloumos/scraper/blob/scraperv2/scraper/config.py#L58-L124), but I don't consider it a must for now.

Reply:
> you can also import `BASE_DIR` from `config.conf`.
> 
> Additionally, if you wanted to do the config a bit more flexible you could structure it like [this one here](https://github.com/kouloumos/scraper/blob/scraperv2/scraper/config.py#L58-L124), but I don't consider it a must for now.

Okay, thanks @kouloumos